src/app/config-page/config-page.component.spec.ts

```diff
@@ -139,9 +140,69 @@ describe('ConfigPageComponent', () => {
     fixture = TestBed.createComponent(ConfigPageComponent);
     component = fixture.componentInstance;
     fixture.detectChanges();
+    configServiceStub = TestBed.get(ConfigService);
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should not save config changes if save is not pressed', done => {
+    fixture.detectChanges();
+    expect(component.displayedButtons).not.toBe(configServiceStub.displayedButtons);
+    const allTabs = fixture.nativeElement.querySelectorAll('div.mat-tab-label');
+    allTabs[2].click();
```

Comment:
Ah...
So would reordering the tabs....
How problematic...

Reply:
I've added a name to the tabs headers so I can access them.  I've had to add it to the span in the template as nothing else goes through to the headers